### DISCUSSION TITLE
Is this repository still maintained? If not, is there a recommended successor?

### DISCUSSION THREAD

**Discussion by programmdesign** [mcguirepr89/BirdNET-Pi#1196] [Q&A] [answered]
@mcguirepr89 I was wondering if this great project is still being maintained? If not, is there a recommended successor? 🙏

**Comment by arne1921KF** [comment, marked as answer]
Have a Look here: https://github.com/mcguirepr89

I have moved my Pi to use @Nachtzuster's repository, which they are actively maintaining and developing: https://github.com/Nachtzuster/BirdNET-Pi

I also have a second installation - for bats! Have a look here at @rdz-oss's repo: https://github.com/rdz-oss/BattyBirdNET-Pi/discussions

This needs a mic with very different capabilities, of course. (OT: I am lucky enough to know someone who had a mostly unused WA Echo Touch 2 Pro sitting in their drawer and lend it to me. And I am impressed by the thing and the accompanying smartphone app.)

Sadly, I couldn't so far get the Pi to detect bats.
When I set it up, I immediately moved to the `bb` branch in the BattyBirdNET-Pi repository, which can do both bats AND birds. If I could get it to work, that is. Currently, I feel like Bilbo in the Fellowship ('stretched'...), but if I find the time to dive in more deeply and if I get it to work I would be thrilled. Especially, of course, if someone would merge some of the ideas from both repos together. 

There is another project I became aware of: https://github.com/tphakala/birdnet-go

I did not look into it, but the repo claims that it could run on a Raspi 3 B. My cursory glance made me think that this isn't as approachable as BirdNet-Pi and BattyBirdNET-Pi, but it is actively maintained and not restricted to Raspberry OS, which makes it a good candidate to gain a larger audience, maybe?
